### PR TITLE
Python-simpleconf initial build 

### PR DESCRIPTION
**Pull request by jayramr**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/simpleconf/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+{% set name = "python-simpleconf" %}
+{% set version = "0.5.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/python-simpleconf-{{ version }}.tar.gz
```
Comment (ndevenish) [inline recipes/simpleconf/meta.yaml:9]: Did you have issues when this was `{{ name }}`?

File: recipes/simpleconf/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+requirements:
+  host:
+    - pip
+    - python >=3.6,<4.0
+    - poetry >=0.12
+  run:
+    - diot >=0.1,<0.2
+    - python >=3.6,<4.0
+    - poetry >=0.12
```
Comment (ndevenish) [inline recipes/simpleconf/meta.yaml:25]: Is poetry required at runtime, or it just install?

(I can't seem to view the CI runs today to see what the issue was)

File: recipes/simpleconf/meta.yaml (hunk 1 above)
Comment (jayramr) [reply recipes/simpleconf/meta.yaml:9]: no issues

File: recipes/simpleconf/meta.yaml (hunk 2 above)
Comment (jayramr) [reply recipes/simpleconf/meta.yaml:25]: poetry required at the building stage. 
let me remove this from run section

File: recipes/simpleconf/meta.yaml (hunk 1 above)
Comment (ndevenish) [reply recipes/simpleconf/meta.yaml:9]: Ah, okay, I see this is how Grayskull left it anyway.

**Comment by ndevenish** [comment]
So the only thing I can see is that I don't know if the repo-creation-bot relies on the recipe folder name (`simpleconf/` here) to be the same as the package name (`python-simpleconf`). Beyond that question, it looks about ready to merge in (which will need to find someone with permissions)

**Comment by jayramr** [comment]
> So the only thing I can see is that I don't know if the repo-creation-bot relies on the recipe folder name (`simpleconf/` here) to be the same as the package name (`python-simpleconf`). Beyond that question, it looks about ready to merge in (which will need to find someone with permissions)

hmm, that would be a concenrn then. Do i need to change the recipe folder. Let's see if the bot fails or figure it properly

**Comment by ndevenish** [comment]
I think whoever actually comes to merge this in (I don't have this power) will know this

**Comment by ndevenish** [comment]
On consideration, changing the folder name to match definitely wouldn't be wrong (the question is whether this would be acceptable) - and if it makes it ready to merge the next time someone has time to review PRs then that would probably be preferable to you.

So I'd say changing to match is probably best.

**Review by viniciusdc — APPROVED** [review]
LGTM

**Comment by viniciusdc** [comment]
[Here](https://github.com/conda-forge/python-simpleconf-feedstock) is the feedtock, might take rougly 40min to show up on conda

**Comment by viniciusdc** [comment]
Hi, @jayramr to update you on this, the above feedstock was already created using conda-smithy, so you will need to request the current maintainer for the above feedstock to add you in as a new maintainer as well.

**Comment by ndevenish** [comment]
Okay, it too me a minute to disentangle what you mean and what happened here:
- This PR made
- Four hours later, the [other PR](https://github.com/conda-forge/staged-recipes/pull/18920) made, also apparently via Grayskull (hence the runtime poetry requirement)
- 18th May, that merged (or, it's successor - there was a new PR for every lint)
- 20th May, this merged

So, the repo was made, around the same time, which I guess caused the confusion here

**Comment by viniciusdc** [comment]
> Okay, it too me a minute to disentangle what you mean and what happened here:
> 
> * This PR made
> * Four hours later, the [other PR](https://github.com/conda-forge/staged-recipes/pull/18920) made, also apparently via Grayskull (hence the runtime poetry requirement)
> * 18th May, that merged (or, it's successor - there was a new PR for every lint)
> * 20th May, this merged
> 
> So, the repo was made, around the same time, which I guess caused the confusion here

yup, but the linter should've caught those alterations (it looks for that package in the repositories list to avoid this) but because of the timing it passed. But it is not anyone's fault, we are working on adding more flexibility to the bot to avoid this in the future. Sorry for the confusion.

**Comment by ndevenish** [comment]
yeah, technically when the PR testing ran it wasn't an existing package :)

**Pull request merged**